src/styles/blocks/product__card.scss

```diff
@@ -0,0 +1,124 @@
+.product__card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid rgb(243, 243, 243);
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  transition: all 0.3s ease-in-out;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .product__name {
+    color: #34568b;
+  }
+
+  .product__image {
+    margin: 32px 20px 40px 20px;
+    height: $product-image-height;
+    background: $mac-img center no-repeat;
+    background-size: contain;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to avoid using shorthand properties for background if only one or two values are set. In this case, you can split 'background' into 'background-image', 'background-position', and 'background-repeat'.